README.md

```diff
@@ -48,298 +48,400 @@ The FWS Payments contract enables ERC20 token payment flows through "rails" - au
 
 - **Account**: Represents a user's token balance and locked funds
 - **Rail**: A payment channel between a client and recipient with configurable terms
-- **Validator**: An optional contract that validates and, if necessary, mediates payment disputes
+- **Validator**: An optional contract that acts as a trusted "arbitrator". It can:
+  - Validate and modify payment amounts during settlement.
+  - Veto a rail termination attempt from any party by reverting the `railTerminated` callback.
+  - Decide the final financial outcome (the total payout) of a rail that has been successfully terminated.
 - **Operator**: An authorized third party who can manage rails on behalf of clients
 
 ### Account
 
 Tracks the funds, lockup, obligations, etc. associated with a single “owner” (where the owner is a smart contract or a wallet). Accounts can be both *clients* and *service providers* but we’ll often talk about them as if they were separate types.
 
 - **Client —** An account that *pays* a service provider (also referred to as the *payer*)
-- **Service Provider** — An account managed the provider of some service which receives payment from a client (also referred to as the *payee*).
+- **Service Provider** — An account managed by the provider of some service which receives payment from a client (also referred to as the *payee*).
```

Comment:
nit: To be honest the notion of client and provider feels secondary to the Payer Payee concept.  Services and clients are an application of payments and mixing that into the terminology here has always felt confusing.  If you agree and are feeling inspired you could swap the primary and secondary names around.